source/sender.ts

```diff
@@ -116,6 +117,12 @@ async function manageMessage(
       factor: 1.3,
       maxRetryTime: 4000,
       async onFailedAttempt(error) {
+        events.dispatchEvent(
+          new CustomEvent("failed-attempt", {
```

Comment:
Should we namespace these events, e.g.,`webext-messenger:failed-attempt` to avoid potential conflict?

Reply:
There's no conflict, the events are dispatched on a specific object. To listen to them, we have to import `events` and use `addEventListener()` on it